tests/cucumber/acceptance/pageObjects/MyCourse.js

```diff
@@ -0,0 +1,17 @@
+const { config } = require("../../../../config");
+class MyCourse {
```

Comment:
will there be any pblm with 
```suggestion
export class MyCourse {
```